build.gradle

```diff
@@ -76,6 +76,7 @@ dependencies {
     implementation "com.sun.xml.bind:jaxb-core:2.3.0.1"
     implementation "com.sun.xml.bind:jaxb-impl:2.3.3"
     implementation "javax.activation:activation:1.1.1"
+    implementation 'com.warrenstrange:googleauth:1.1.2'
```

Comment:
Have you done any analysis of available libraries? It looks like this one hasn't been updated for a while. Also, have you considered embedding implementation? It looks like it's a really simple algorithm that can be implemented easily on our side.

Reply:
I did not consider it as back when I needed it I needed it fast. And using existing code was faster then writing it myself.

Reply:
OK, can you please investigate this.